django/contrib/sessions/backends/cache.py

```diff
@@ -62,14 +64,15 @@ def save(self, must_create=False):
             raise CreateError
 
     def exists(self, session_key):
-        return session_key and (KEY_PREFIX + session_key) in self._cache
+        return session_key and (self.cache_key_prefix +
```

Comment:
one longer line is preferred (or at least avoid non-multiple of four indentation).